CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release Notes
 
```

Comment:
Easier to add it now since we have the template in the diff, haha.

```suggestion

## X.X.X (YYYY-MM-DD)

TODO


```

Reply:
I'm on master.

Reply:
crap LOL thought this was a proper release-dedicated branch thingy. Nice catch.